toolbox/README.md

```diff
@@ -0,0 +1,28 @@
+#  Rook Toolbox
+The rook toolbox is a container with common tools used for rook debugging and testing.  All packages in the toolbox can be seen in the [Dockerfile](Dockerfile)
+
+## Container Linux by CoreOS
+To use the rook toolbox on CoreOS, first add the following values to the toolbox config file:
+```
+cat >~/.toolboxrc <<EOL
+TOOLBOX_DOCKER_IMAGE=quay.io/rook/toolbox
+TOOLBOX_DOCKER_TAG=latest
+EOL
+```
+
+Then launch the toolbox as usual:
+```
+toolbox
+```
+
+### Ceph Tools
+To use the ceph tools from a rook host, launch the toolbox and run the following command:
+```
+toolbox --bind=/var/lib/rook:/var/lib/rook
+ROOK_CONFIG=`find /var/lib/rook -regex '.*mon[0-9]+/.*\.config' | head -1`
```

Comment:
i'm not sure there's a super compelling reason for us to have to change our config file name.  With the latest commit the toolbox does everything for a user of it, so they don't even have to think about the config file: https://github.com/rook/rook/pull/381/commits/fb70cb9f335268d0cbb7ae331c954b9fa0b8baa3